src/Player.java

```diff
@@ -0,0 +1,50 @@
+
+public class Player extends User { 
+	/*
+	 * A Player will be an User that will play the game and manage its team and items.
+	 */
+	
+	//Attributes
+	private ABML<item> inventory; //Collections of the items that the player has.
+	private PlayableCharacter team; //Character that the Player will use.
+	//private ABML<PlayableCharacter team; //If there's time, there'll be a Collections of characters that the Player could use.
+	private int gold; //Amount of gold the Player will have
+	
+	//Constructor
+	public Player() {}
+
+	
+	//Getters and Setters
+	public ABML<item> getInventory() {
+		return inventory;
+	}
+
+	public void setInventory(ABML<item> inventory) {
+		this.inventory = inventory;
+	}
+
```

Comment:
I think you shouldn't send as parameter a full list of thing to set, so inventory shouldn't have a setter, but methods to add and remove single items to the inventory